projects/1.20/assets/the-dawn-era/dawnera/lang/zh_cn.json

```
@@ -0,0 +1,200 @@
+{
+  "accessibility.dawnera.private": "private",
```

```suggestion
  "accessibility.dawnera.private": "Private",
```
既然汉化会崩端，那么请保持原文
此类问题请全文通查并修改

---

> 既然汉化会崩端，那么请保持原文 此类问题请全文通查并修改

现在最新版已修复